build/rewrite.go

```diff
@@ -64,31 +60,49 @@ func allowedSort(name string) bool {
 // nil, a default set of rewrites will be used that is determined by
 // the type (BUILD vs default starlark) of the file being rewritten.
 type Rewriter struct {
-	DisableRewrites                 *[]string
-	IsLabelArg                      *map[string]bool
-	LabelDenyList                   *map[string]bool
-	IsSortableListArg               *map[string]bool
-	SortableDenylist                *map[string]bool
-	SortableAllowlist               *map[string]bool
-	NamePriority                    *map[string]int
-	StripLabelLeadingSlashes        *bool
-	ShortenAbsoluteLabelsToRelative *bool
+	RewriteSet                      *map[string]struct{}
```

Comment:
Changing to get rid of pointer. I'm curious as to if changing it to a slice would change it besides adding in a loop to go through the set?

Reply:
Well, it makes declaring it more straightforward, you just create a slice with the names of the rewrites instead of having to set entries in a map where the value is unused. If someone that's more knowledgeable about go says that using a map as a set like this is idiomatic code, then feel free to stick with it.

Reply:
Gotcha. I'm not entirely convinced it's more straightforward to initialize it as a slice vs map. Both seem viable here. I would argue that it's more intuitive for checking if something exists by checking for a key rather than adding a for loop through the slice. I'm going to leave it as is unless someone has a strong opinion on this.

Reply:
I definitely think it's more straightforward to say rewriteSet = []string{"callsort", "loadsort"} than having to initialize a map with values that are unused, but if sticking with the map, can you make the value type interface{} and then the value can just be nil. There should also be a comment indicating that the values are ignored.

Reply:
Okay, I ended up changing it back to a slice :)